### PR TITLE
Makes rescaling more reliable in tests.

### PR DESCRIPTION
**Pull request by jdoliner**

### CODE REVIEW

**Comment by jdoliner** [comment]
@derekchiang whenever you have a second, this PR should hopefully make our CI more stable.

**Comment by jdoliner** [comment]
@derekchiang could you take a look at this today? Would like to merge it in before too many conflicts accrue.

File: src/server/pachyderm_test.go [hunk 1]
```diff
@@ -2089,11 +2083,26 @@ func scalePachd(t *testing.T, k *kube.Client) {
...
+
+func waitForReadiness(t *testing.T) {
+	k := getKubeClient(t)
+	for {
+		has, err := kube.ControllerHasDesiredReplicas(k, pachdRc(t))()
+		require.NoError(t, err)
+		if has {
+			break
+		}
+		time.Sleep(time.Second * 30)
```
Comment (derekchiang) [inline src/server/pachyderm_test.go]: You sure we wanna sleep for this long?

**Comment by derekchiang** [comment]
LGTM other than potentially changing the sleep time.

File: src/server/pachyderm_test.go (hunk 1 above)
Comment (jdoliner) [inline src/server/pachyderm_test.go]: We could make it less you think 10 seconds sounds good?

**Pull request merged**